client/test/e2e/services/services.spec.js

```diff
@@ -0,0 +1,106 @@
+/*global describe, it, element, by, beforeEach, inject, browser */
+
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+var expect = chai.expect;
```

Comment:
If you have the latest master, you can do this:
```js
var chai = require('chai');
var expect = chai.expect;
var helpers = require('../shared/helpers');

// this will bind chaiAsPromised automatically for you
helpers.configure(chai);
```